providers/azure.go

```diff
@@ -235,11 +238,35 @@ func getEmailFromJSON(json *simplejson.Json) (string, error) {
 	return email, err
 }
 
+func getEmailFromIDToken(idToken string) (string, error) {
+	jwt := strings.Split(idToken, ".")
+	jwtData := strings.TrimSuffix(jwt[1], "=")
+	b, err := base64.RawURLEncoding.DecodeString(jwtData)
+	if err != nil {
+		return "", fmt.Errorf("jwt is mailformed: %w", err)
```

Comment:
Typo in "malformed".

Reply:
A mailformed jwt should be fine in `getEmailFromIDToken`, right? ;)

Reply:
I thought of this, but then if `jwt` is _mailformed_ why does is it appear in an error message inside `getEmailFromIDToken` :man_shrugging:

Reply:
good eyes!